rosdep/python.yaml

```diff
@@ -1367,6 +1367,13 @@ python-numpy:
     vivid: [python-numpy]
     wily: [python-numpy]
     xenial: [python-numpy]
+python-numpy-stl:
```

Comment:
The key should have a `-pip` suffix to leave room for this name if the package becomes available through Debian packages in the future.